edx/edx-enterprise
# ENT-191 Fix package listing in setup.py to avoid import errors

**Description:** The LMS was 500ing due to import errors when pulling in the latest version of this package because integrated_channels was not added in the list of packages in setup.py.

**JIRA:** https://openedx.atlassian.net/browse/ENT-191

**Dependencies:** 

**Merge deadline:** List merge deadline (if any)

**Installation instructions:** List any non-trivial installation 
instructions.

**Testing instructions:**

1. In devstack on bexline/sf_oauth for edx-platform, start the LMS server. Try hitting any page of the devstack server, it should 500 with an import error.
2. Change requirements/edx/base.txt in edx-platform to comment out the line "edx-enterprise==0.26.1" and add the line "git+https://github.com/edx/edx-enterprise.git@9c4f782a26570c60b098b1d3346da8ef0513a5ff#egg=edx-enterprise==0.26.2".
3. Run paver install_python_prereqs
4. Restart the lms server on devstack, and try hitting any page. It should be successful.

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**

**Author concerns:** List any concerns about this PR - inelegant 
solutions, hacks, quick-and-dirty implementations, concerns about 
migrations, etc.
